src/web/app/pages-instructor/instructor-student-activity-logs/instructor-student-activity-logs.component.spec.ts

```diff
@@ -310,15 +258,17 @@ describe('InstructorAuditLogsPageComponent', () => {
       searchFrom: (new Date('2020-12-31T00:00+00:00').getTime() - tzOffset * 60 * 1000).toString(),
       searchUntil: (new Date('2021-01-01T00:00+00:00').getTime() - tzOffset * 60 * 1000).toString(),
       studentEmail: testStudent.email,
+      sessionName: '',
+      logType: 'submission',
     });
 
     expect(component.searchResults.length).toEqual(2);
 
     for (let i: number = 0; i < 2; i += 1) {
-      expect(component.searchResults[i].isTabExpanded).toBeFalsy();
+      expect(component.searchResults[i].isTabExpanded).toBeTruthy();
       expect(component.searchResults[i].logColumnsData).toEqual(resultColumns);
       // Testing that the LogType is converted correctly.
-      expect(component.searchResults[i].logRowsData[0][2].value).toEqual('Submitted responses');
+      expect(component.searchResults[i].logRowsData[0][0].value).toContain('Submitted responses');
```

Comment:
Why is `toEqual` no longer working here?

Reply:
@wkurniawan07 the whole message is `Submitted responses at <timestamp>`, but we dont have access to the timestamp so we cant compare the whole message. Plust I think using `toContain` is enough to check LogType conversion

Reply:
> we dont have access to the timestamp
I don't believe this is the case. The test data is fully within our control.